test/org/redhatcop/util/files/params/notifications-build.params

```diff
@@ -0,0 +1,5 @@
+APPLICATION_NAME=notifications-test
+NAMESPACE=pipeline-library-test
+SOURCE_REPOSITORY_URL=https://github.com/etsauer/pipeline-library.git
```

Comment:
Same as above, but in this case it can be removed as the default in template is `https://github.com/redhat-cop/pipeline-library.git`

Reply:
yes